assignments/this.js

```diff
@@ -13,14 +13,72 @@
 
 // code example for Window Binding
 
+function windowBinding() {
+    return this;
+}
+
+console.log(windowBinding());
+
 // Principle 2
 
 // code example for Implicit Binding
 
+const animal =  {
+    species: "dog",
+    about: function(name) {
+        console.log(`${this.name} is a ${this.species}`);
+    }
+};
+
+animal.about("Tucker");
+
```

Comment:
You don't need the this keyword to point to a variable since the variable will be passed in outside the object.. <h3>console.log(`${name} is a ${this.species}`);</h3> should work!